ARK Server Manager/Lib/Model/ServerProfile.cs

```diff
@@ -1889,5 +1896,25 @@ public void ResetOverrideMaxExperiencePointsDino()
         {
             OverrideMaxExperiencePointsDino = GameData.DEFAULT_MAX_EXPERIENCE_POINTS_DINO;
         }
+
+        // section reset methods
+        public void ResetChatAndNotificationSection()
+        {
+            EnableGlobalVoiceChat = GameData.DEFAULT_ENABLE_GLOBAL_VOICE_CHAT;
```

Comment:
Sorry, that's not what I mean.  What I mean is that now there are "two" ways defaults get set: one is from property metadata, the other is through these functions (maybe later there will be others.)  A better pattern might be to define a function that resets the property to the value declared in the property metadata, like what the code at this link does: http://bursjootech.blogspot.com/2008/01/reset-dependencyproperty-to-its-default.html